### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by VerisimilitudeX** [pigfyy/mosachi#1]
@pigfyy do you own mosachi.org?

**Comment by pigfyy** [comment]
nope, friend of mine owns it and we wanted to collaborate on the project further in the future, so I recoded the project basically from scratch, aside from some of the data-processing logic to help make it more maintainable. 
repo for the old project (also the one hosted by the mosachi.org domain): https://github.com/DumplingLife/mosachi-rewrite

**Comment by VerisimilitudeX** [comment]
> nope, friend of mine owns it and we wanted to collaborate on the project further in the future, so I recoded the project basically from scratch, aside from some of the data-processing logic to help make it more maintainable. repo for the old project (also the one hosted by the mosachi.org domain): https://github.com/DumplingLife/mosachi-rewrite

cool! i've been using mosachi for the last couple of months and I really like it. i do have some ideas on how to improve it, so do you know how i should proceed? i would like to contribute to mosachi in the long term.

**Comment by VerisimilitudeX** [comment]
@pigfyy
